Package.swift

```diff
@@ -15,7 +15,7 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         .package(url: "https://github.com/evgenyneu/keychain-swift.git", from: "20.0.0"),
-        .package(url: "https://github.com/awslabs/aws-sdk-swift", from: "1.0.0")
```

Comment:
Nit: It's not really necessary to bump this number, it will grab the latest minor version available due to the "from:" declaration.